canvas_modules/common-canvas/src/common-properties/properties-main/properties-main.jsx

```diff
@@ -419,11 +426,19 @@ class PropertiesMain extends React.Component {
 		this.setState({ showPropertiesButtons: state });
 	}
 
+	updateRightFlyoutWidth(size) {
```

Comment:
This seems like the wrong place for setting this.  Why is this needed here?

Reply:
Since we are trying to implement resize in `cc-right-flyout` which will resize `right-flyout-panel` as per drag width. So when we click on panel resize it will only resize the common properties content and will not expand/shrink the right-flyout-panel so to enforce width on CC right-flyout I have included this. Please let me know if you can suggest another way of doing this.

Without enforcing width to right-flyout in properties it will look something like below, where the properties content width is set as per custom classnames but flyout width is not reset.

<img width="980" alt="Screenshot 2024-09-23 at 12 19 42 PM" src="https://github.com/user-attachments/assets/0716f94c-20da-459f-8b2b-101f413329f8">

Also could you please where can I find any reference to feature flag in the code ?